test/Esprima.Tests/Fixtures/es2018/dynamic-import/await-import.tree.json

```diff
@@ -1,422 +1,148 @@
-                                        }
-                                    }
-                                ],
-                                "optional": false,
-                                "range": [
-                                    28,
-                                    37
-                                ],
-                                "loc": {
-                                    "start": {
-                                        "line": 1,
-                                        "column": 28
-                                    },
-                                    "end": {
-                                        "line": 1,
-                                        "column": 37
-                                    }
-                                }
-                            },
-                            "range": [
-                                22,
-                                37
-                            ],
-                            "loc": {
-                                "start": {
-                                    "line": 1,
-                                    "column": 22
-                                },
-                                "end": {
-                                    "line": 1,
-                                    "column": 37
-                                }
-                            }
-                        },
-                        "range": [
-                            22,
-                            38
-                        ],
-                        "loc": {
-                            "start": {
-                                "line": 1,
-                                "column": 22
-                            },
-                            "end": {
-                                "line": 1,
-                                "column": 38
-                            }
-                        }
-                    }
-                ],
-                "range": [
-                    20,
-                    39
-                ],
-                "loc": {
-                    "start": {
-                        "line": 1,
-                        "column": 20
-                    },
-                    "end": {
-                        "line": 1,
-                        "column": 39
-                    }
-                }
-            },
-            "generator": false,
-            "expression": false,
-            "async": true,
-            "range": [
-                0,
-                39
-            ],
-            "loc": {
-                "start": {
-                    "line": 1,
-                    "column": 0
-                },
-                "end": {
-                    "line": 1,
-                    "column": 39
-                }
-            }
-        }
-    ],
-    "sourceType": "script",
-    "range": [
-        0,
-        39
-    ],
-    "loc": {
-        "start": {
+  "type": "Program",
```

Comment:
Is there a place I can find this file, or is it generated locally based on the test itself?

Reply:
Don't know what you mean...
The AST was different with the old implementation.... Now it is correct (I think :-) )